src/components/Form/FormProvider.js

```diff
@@ -70,8 +70,12 @@ const propTypes = {
     shouldValidateOnBlur: PropTypes.bool,
 
     shouldValidateOnChange: PropTypes.bool,
+
+    shouldSetTouchedOnBlurOnly: PropTypes.bool,
 };
 
+const VALIDATE_DELAY = 200;
```

Comment:
Can you please add a comment to explain why the value is `200` and not any other value?